Content.Shared/LandMines/LandMineComponent.cs

```diff
@@ -1,13 +1,15 @@
 using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
 
-namespace Content.Server.LandMines;
+namespace Content.Shared.LandMines;
 
 [RegisterComponent]
 public sealed partial class LandMineComponent : Component
```

Comment:
Could we get some documentation here?

Reply:
I am admittedly not the best at documentation. Please have a look if the documentation I have added is relevant and helpful.